lib/appmap/service/validator/config_validator.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'appmap/service/validator/violation'
+
+module AppMap
+  module Service
+    module Validator
+      class ConfigValidator
+        attr_reader :config, :violations
+
+        def initialize(config_file)
+          @config_file = config_file
+          @violations = []
+          @config = load_config
+        end
+
+        def valid?
+          validate_config_presence
```

Comment:
Are you going to check the Ruby version as well?

Reply:
Planned to implement in a follow up PR, but it seems to be simple, so adding now.